frontend/src/dialogs/SecretDialogAzure.vue

```diff
@@ -21,7 +21,7 @@ limitations under the License.
     :dataValid="valid"
     :secret="secret"
     cloudProviderKind="azure"
-    color="blue"
+    color="blue darken-1"
```

Comment:
Nothing to change here but we should have constants for infrastructureProvider colors and textColors.